tests/node/rpc/messages/GetVehicleDataResponseTests.js

```diff
@@ -0,0 +1,79 @@
+const SDL = require('./../../../../lib/js/dist/SDL.min.js');
+const GetVehicleDataResponse = SDL.rpc.messages.GetVehicleDataResponse;
+const FunctionID = SDL.rpc.enums.FunctionID;
+const MessageType = SDL.rpc.enums.MessageType;
+const WindowStatus = SDL.rpc.structs.WindowStatus;
+const WindowState = SDL.rpc.structs.WindowState;
+const Grid = SDL.rpc.structs.Grid;
+
+const BaseRpcTests = require('./BaseRpcTests');
+const Test = require('./../../../Test.js');
+const Validator = require('./../../../Validator.js');
+
+
+describe('GetVehicleDataResponseTests', function () {
+    before(function () {
+        const windowState = new WindowState()
```

Comment:
In Test.JS, 	can you set up generic Grid and generic WindowState objects which you can reuse in your test classes?  So instead of instantiating these objects in the Test classes, you can just set them to the Test objects.

In Android, it is done like this (in TestValues.java):
public static final Grid GENERAL_LOCATION_GRID 				  = new Grid(2, 3);
public static final WindowState GENERAL_WINDOW_STATE 		  = new WindowState(2, 3);

example usage in WindowStatusTests.java:
msg.setLocation(TestValues.GENERAL_LOCATION_GRID); // references TestValues.java

Reply:
@santhanamk done